index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <header class="bar-nav">
+        <a href="index.html" class="logo">My Logo</a>
+        <nav><img src="image/Union.png" alt="Menu icon"></nav>
+      </header>
+      <div class="container">
+        <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
+        <section class="detail two">
```

Comment:
- [ ] Let's make sure our introduction paragraph is a `p` tag for better HTML semantic

Reply:
I have made changes to it